libraries/AC_Avoidance/AP_OADijkstra.cpp

```diff
@@ -395,20 +401,18 @@ bool AP_OADijkstra::calc_shortest_path(const Location &origin, const Location &d
     update_visgraph(_source_visgraph, {AP_OAVisGraph::OATYPE_SOURCE, 0}, origin_NE, true, destination_NE);
     update_visgraph(_destination_visgraph, {AP_OAVisGraph::OATYPE_DESTINATION, 0}, destination_NE);
 
-    // clear short_path_data array
-    _short_path_data_numpoints = 0;
+    // check OA_DIJKSTRA_POLYGON_SHORTPATH_NOTSET_IDX is defined correct
+    static_assert(OA_DIJKSTRA_POLYGON_SHORTPATH_NOTSET_IDX > OA_DIJKSTRA_POLYGON_FENCE_PTS, "check OA_DIJKSTRA_POLYGON_SHORTPATH_NOTSET_IDX > OA_DIJKSTRA_POLYGON_FENCE_PTS");
 
     // add origin and destination (node_type, id, visited, distance, distance_from_idx) to short_path_data array
```

Comment:
Comment needs to be changed to be according to new struct order.

Reply:
sorry, I don't know what this means..

Reply:
fixed, thanks!